lib/dish.rb

```diff
@@ -0,0 +1,7 @@
+class Dish
+  attr_reader :name, :price
+  def initialize (name, price)
```

Comment:
Style/SpaceAfterMethodName: Do not put a space between a method name and the opening parenthesis. (https://github.com/bbatsov/ruby-style-guide#parens-no-spaces)